alot/buffers.py

```diff
@@ -85,7 +85,7 @@ def rebuild(self):
             self.isinitialized = True
 
         lines = list()
-        displayedbuffers = filter(self.filtfun, self.ui.buffers)
```

Comment:
I'm OK with eliminating lambdas as you do. Where we use already defined functions to filter I'd rather use them in combination with filter instead of list comprehensions.
But this is really rather based on my personal stylistic preferences.
@lucc: you decide